src/sections/wallet/strategy/NewDepositForm/NewDepositAssetSelector.utils.ts

```diff
@@ -64,11 +66,13 @@ export const useNewDepositDefaultAssetId = (assetId?: string) => {
 
       if (reserveAccountBalance) return reserveAccountBalance.id
 
-      const ethBalance =
-        accountAssets.accountAssetsMap.get(ETH_ASSET_ID)?.balance
-          ?.freeBalance ?? "0"
+      for (const prefAssetId of GETH_PREFERRED_ASSETS) {
```

Comment:
it's not necessary here, since we already iterate pool reserves as WSTETH_ASSET_ID and ETH_ERC20_ASSET_ID above (reserveAccountBalance)